zone/client_packet.cpp

```diff
@@ -8927,12 +8927,17 @@ void Client::Handle_OP_ItemVerifyRequest(const EQApplicationPacket *app)
 				{
 					if (item->RecastDelay > 0)
 					{
-						if (!GetPTimers().Expired(&database, (pTimerItemStart + item->RecastType), false)) {
+						if (item->RecastType != -1 && !GetPTimers().Expired(&database, (pTimerItemStart + item->RecastType), false)) {
 							SendItemRecastTimer(item->RecastType); //Problem: When you loot corpse, recast display is not present. This causes it to display again. Could not get to display when sending from looting.
 							MessageString(Chat::Red, SPELL_RECAST);
 							SendSpellBarEnable(item->Click.Effect);
 							LogSpells("Casting of [{}] canceled: item spell reuse timer not expired", spell_id);
 							return;
+						} else if (item->RecastType == -1  && !GetPTimers().Expired(&database, (pTimerNegativeItemReuse * item->ID), false)) {
```

Comment:
For all of these checks that do `recast_type == -1` can we swap them out for a constant for readability? We've been making sincere efforts to remove magic numbers from the source for the past few years.

This makes code more readable in its intent for new travelers

Somewhere in say `eq_constants.h`

```cpp
constexpr uint16 RECAST_TYPE_ITEM = -1;
```

Then in code

```cpp
} else if (item->RecastType == RECAST_TYPE_ITEM ...
```

Reply:
-1 maybe UNLINKED_ITEM_TIMER  ?  because anything other than -1 is a linked timer

Reply:
Or `RECAST_TYPE_ITEM_UNLINKED` with a description in comments at the end of the constant

The constant is not a timer so might be confusing

Reply:
Pretty sure I got all the spots where this is referenced